docs/src/pages/_app.page.tsx

```diff
@@ -8,6 +9,7 @@ import { Header } from '@/components/Layout/Header';
 import Script from 'next/script';
 import { baseTheme } from '../theme';
 import { useCustomRouter } from '@/components/useCustomRouter';
+import { awsExports } from '../aws-exports.js';
```

Comment:
Where is this file 😃

Reply:
It is in the root of the src folder

Reply:
Maybe I'm missing something but I don't see it in the PR?

Reply:
Yeah. That is strange because I have made multiple commits since that suggestion.

Reply:
Was it gitignored?

Reply:
I'm not sure but I have added it now